CHANGELOG.md

```diff
@@ -7,7 +7,8 @@
 - Use [VideoIO](https://github.com/JuliaIO/VideoIO.jl) for faster rendering without temporary images
 - Ability to draw text in an animated way
 - Ability to morph with `fill` or `stroke` and using `SubAction` to specify changes in color
-  
+- Added live viewer based on `Gtk.jl`
+- Prototyped ability to return single frame of Javis animation
```

Comment:
Here as well: Name the function. Explain the user how to use it. Yes I should do this for `Ability to draw text in an animated way` as well